rails_event_store/lib/rails_event_store/after_commit_async_dispatcher.rb

```diff
@@ -0,0 +1,50 @@
+module RailsEventStore
+  class AfterCommitAsyncDispatcher
+    def initialize(scheduler:)
+      @scheduler = scheduler
+    end
+
+    def call(subscriber, _, serialized_event)
+      run(->() { @scheduler.call(subscriber, serialized_event) })
```

Comment:
```
run{ @scheduler.call(subscriber, serialized_event) }
```

Reply:
Notice that if scheduler call API was identical to dispatcher's API, you could simply substitute scheduler for another dispatcher. Wouldn't that be nice?

```
AfterCommitAsyncDispatcher.new(ActiveJobDispatcher.new)
```